gammapy/maps/geom.py

```diff
@@ -484,6 +484,8 @@ def from_nodes(cls, nodes, **kwargs):
         nodes = np.array(nodes, ndmin=1)
         if len(nodes) < 1:
             raise ValueError("Nodes array must have at least one element.")
+        if len(nodes) != len(np.unique(nodes)):
+            raise ValueError("Node values must be unique")
```

Comment:
I would write "MapAxis : node values must be unique" to help the user understand the origin of the issue

Reply:
In the traceback you see where the error comes from...no?